openedx_lti_tool_plugin/models.py

```diff
@@ -241,7 +240,7 @@ class LtiGradedResource(models.Model):
         related_name='openedx_lti_tool_plugin_graded_resource',
         help_text=_('The LTI profile that launched the resource.'),
     )
-    context_key = LearningContextKeyField(
+    context_key = models.CharField(
```

Comment:
Why the `validate_context_key` validator is defined for migration but not for model field?. @alexjmpb

Reply:
I missed to create the migrations, however I'm gonna apply the previous comment